cpp/src/barretenberg/honk/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-barretenberg_module(honk numeric ecc srs proof_system transcript)
+barretenberg_module(honk numeric ecc srs proof_system transcript plonk)
```

Comment:
I'm assuming this is temporary for the proving key or something. Worth leaving a TODO or is this part of your immediately forthcoming work?

Reply:
Yeah, I'll make a note of that, thanks.